timescaledb/how-to-guides/toolkit/counter_agg.md

```diff
@@ -0,0 +1,164 @@
+# Counter aggregates
+There are two types of metrics that can be recorded in a database: gauges, and
+counters. Gauges fluctuate up and down, like temperature or speed, while
+counters always increase, like the total number of visitors to a website.
+
+When you process counter data, it is usually assumed that if the value of the
+counter goes down, the counter has been reset. For example, if you wanted to
+count how many times a runner ran around a track, you would expect the values to
+continuously increase: 1, 2, 3, 4, and so on. If the counter reset to 0, you
+would expect that this was a new runner starting on their first lap, not the
+original runner going back to 1. This can become a problem if you want to
+continue counting from where you left off, rather than resetting to 0. A reset
+could occur if you have had a short server outage, or any number of other
+reasons. To get around this, you can analyze counter data by looking at the
```

Comment:
I think introducing the second runner is confusing here? Perhaps we could continue the car analogy? Where if someone were able to reset the odometer (maybe using the "TRIP" button, you can still calculate the overall miles traveled by assuming that each time it resets or decreases, you just keep a running total and treat each time it decreases as a reset. I'm not expressing this well, but maybe it'll spark something for you...